views/qrcode-results.pug

```diff
@@ -0,0 +1,28 @@
+doctype html
+html
+  head
+    title Live Coding Train Poll
+    link(rel="stylesheet" href="css/style.css")
+    link(rel="stylesheet" href="css/qrcode.css")
+    script(src="/js/qrcode.js")
+
+  body
+    h1 The Coding Train Live Poll
+    h2 QR-Code to newest Poll
+    section.content
+      section.left
+        include poll.pug
+      section.right
+        h3.qr-question=question
+        div#placeHolder
+        template(data-pollurl=pollURL)
```

Comment:
should be:
```
  body
    h1 The Coding Train Live Poll
    section.content
      section.left
        include poll.pug
      section.right
        div#placeHolder
    template(data-pollurl=pollURL)
```

That cleansup the look by a loot and removes repeating headlines / informations with wrong stylings.